maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationServiceTest.java

```diff
@@ -73,6 +73,7 @@ void givenARequestWithC3Enabled_whenCreateIsInvoked_thenApplicationDTOIsUpdatedW
         verify(maatCourtDataService).invokeStoredProcedure(applicationDTO, workflowRequest.getUserDTO(),
                                                            StoredProcedure.ASSESSMENT_POST_PROCESSING_PART_2
         );
+        verify(assessmentSummaryService, times(1)).getSummary(any());
```

Comment:
Minor, but I think we should specify the expected type if possible, rather than `any()`.